package.json

```diff
@@ -121,7 +123,6 @@
     "eslint-plugin-verdaccio": "9.6.1",
     "fs-extra": "9.1.0",
     "get-stdin": "8.0.0",
-    "husky": "2.7.0",
```

Comment:
If this was removed because it didn't work with PnP then `husky@>=4` supports PnP

Reply:
Thanks for the tip 💯 

😢 I'm struggling with eslint, everything else works fine. https://github.com/yarnpkg/berry/issues/8

Reply:
You can try applying this to your eslint config https://github.com/yarnpkg/berry/blob/1462caa87dbccb7e720194d768c60871de903911/packages/eslint-config/index.js#L1-L2

Reply:
Actually that's not the issue, here is a diff that fixes most of the errors
```diff
diff --git a/tsconfig.json b/tsconfig.json
index 71292d23..8de2a56f 100644
--- a/tsconfig.json
+++ b/tsconfig.json
@@ -9,12 +9,7 @@
     "resolveJsonModule": true,
     "outDir": "lib",
     "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true,
-    "typeRoots": [
-      "./node_modules/@verdaccio/types/lib/verdaccio",
-      "./node_modules/@types",
-      "./types/custom.d.ts"
-    ]
+    "esModuleInterop": true
   },
   "exclude": [
     "node_modules",
diff --git a/types/custom.d.ts b/types/custom.d.ts
index 9c7e201c..35f81819 100644
--- a/types/custom.d.ts
+++ b/types/custom.d.ts
@@ -1,3 +1,5 @@
+/// <reference types="@verdaccio/types" />
+
 import { Logger, RemoteUser } from "@verdaccio/types";
 
 declare global {

```

Reply:
awesome :) I was not even looking there thanks !

Reply:
For the `Resolve error: unable to load resolver "node"  import/order` errors the issue is you're missing a dependency on `eslint-import-resolver-node`
```
yarn add eslint-import-resolver-node
```

Reply:
Thanks :) you have saved me hours looking in the wrong place.

Reply:
My pleasure 👍